src/Bot.ts

```diff
@@ -0,0 +1,24 @@
+import { Telegraf, Context } from 'telegraf';
+import i18next from 'i18next';
+
+export default class Bot {
```

Comment:
just to clarify. Will we use default exporting through the project. AFAIK TS autoimport does not like it

Reply:
It's the first time I hear of such issue. Do you have more details? I never experienced any problems with it.

Regarding what should we use, it should depend on a specific case.

Reply:
nono, it is okay, just double-checked it will work property with default importing as well